Migration.md

```diff
@@ -317,9 +301,12 @@ but with improved human-readability..
 
 ### Deprecations
 
+1. **inerial.sdf** `//inertial/pose/@relative_to` attribute is deprecated
```

Comment:
Can you change this to say removed instead of deprecated?

Reply:
Done 655b83ff2b72e4bb8b0279433533c7ccbc2468da